Graph/Basic Graph Traversal/133.py

```diff
@@ -0,0 +1,49 @@
+"""
+# Definition for a Node.
+class Node:
+    def __init__(self, val = 0, neighbors = None):
+        self.val = val
+        self.neighbors = neighbors if neighbors is not None else []
+"""
+
+# BFS Solution
+class Solution:
+    def cloneGraph(self, node: 'Node') -> 'Node':
+        if not node:
+            return node
+
+        graph = {}
```

Comment:
The name `graph` is misleading here. It's not a representation of any graph; it's a mapping between those original vertices and their corresponding cloned ones.